src/schema/paddle.ts

```diff
@@ -429,6 +465,34 @@ export const resolvers: IResolvers<unknown, AuthContext> = traceResolvers<
 
       return consolidated.filter(Boolean) as BasePricingPreview[];
     },
+    pricingPreviewByIds: async (
+      _,
+      { ids, locale }: PaddlePricingPreviewByIdsArgs,
+      ctx,
+    ): Promise<BasePricingWithoutMetadata[]> => {
+      if (!ids?.length) {
+        throw new ValidationError('No ids provided');
+      }
```

Comment:
Not needed since `ids` are required in schema

Reply:
It can be an empty array though.

Reply:
diff marked it wrong I mean 
```suggestion
      if (!ids.length) {
        throw new ValidationError('No ids provided');
      }
```